File: experiment/java-python/test/j2pyrawpy/for__0__CHECK_WHETHER_SECOND_STRING_CAN_FORMED_FIRST_STRING_USING_COUNT_ARRAY.py
#!/usr/bin/env python
""" generated source for module for__0__CHECK_WHETHER_SECOND_STRING_CAN_FORMED_FIRST_STRING_USING_COUNT_ARRAY """
class X(object):
    """ generated source for class X """
    @classmethod
    def canMakeStr2(cls, str1, str2):
        """ generated source for method canMakeStr2 """
        count = [None]*MAX
        str3 = str1.toCharArray()
        str4 = str2.toCharArray()
        i = 0
        while i < str4.length:
            if count[str4[i]] == 0:
                return False
            count[str4[i]] -= 1
            i += 1
        return True

File: experiment_jp_test/mutants/for__0__CHECK_WHETHER_SECOND_STRING_CAN_FORMED_FIRST_STRING_USING_COUNT_ARRAY.java
static boolean canMakeStr2 ( String str1 , String str2 ) { int [ ] count = new int [ MAX ] ; char [ ] str3 = str1 . toCharArray ( ) ; char [ ] str4 = str2 . toCharArray ( ) ; for ( int i = 0 ; i < str4 . length ; i ++ ) { if ( count [ str4 [ i ] ] == 0 ) return false ; count [ str4 [ i ] ] -- ; } return true ; }
